Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'devise-security'
 gem 'kaminari'
 
 gem 'rolify'
+gem 'geocoder', '>= 1.6.1'
```

Comment:
Let me know if there is a more logical location for this in the gemfile 😅

Reply:
i think this is fine for now. 
if more of the `ahoy` gem's dependencies get hit, we can see about updating the actual `ahoy` gem to see if it fixes it